code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -614,7 +614,7 @@
 
 /datum/action/xeno_action/activable/corrosive_acid/use_ability(atom/A)
 	var/mob/living/carbon/xenomorph/X = owner
-	if(!A.dissolvability(acid_type))
+	if(!A.dissolvability(initial(acid_type.acid_strength)))
 		X.balloon_alert(X, "We cant melt this")
 		return fail_activate()
```

Comment:
Is this actually needed? I think can_use_ability is called prior to use_ability anyway.

Reply:
I unno why this is commenting on the outdated code, but point remains.